Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.Layout.cs

```diff
@@ -54,6 +55,27 @@ protected override Size ArrangeOverride(Size finalSize)
             return base.ArrangeOverride(finalSize);
         }
 
+        /// <summary>
+        /// Is the Xaml Content loaded and live?
+        /// </summary>
+        /// <returns>True if the Xaml content is properly loaded</returns>
+        private bool IsXamlContentLoaded()
+        {
+            if (_xamlSource.Content == null)
+            {
+                return false;
+            }
+
+            if (WUX.Media.VisualTreeHelper.GetParent(_xamlSource.Content) == null)
```

Comment:
can you use FrameworkElement.IsLoaded?

Reply:
FrameworkElement.IsLoaded's timing isn't quite right for this, if I recall - but if that'd work that'd be a nice simplification.  Oscar, can you try that?

Reply:
This is out of scope for this change.

---
In reply to: [260108644](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260108644) [](ancestors = 260108644)